open-banking-accelerator/accelerators/ob-apim/README.md

```diff
@@ -0,0 +1,17 @@
+### **WSO2 Open Banking APIM Accelerator 3.0.0**
+
+**Try Locally:**
+
+Prerequisites:
+1. Download WSO2 API Manager product 
```

Comment:
should we include this part in the open source repo? since build packs cant be accessed? maybe we should include readme to build and get the packs created in the specific location?

Reply:
I updated the readme as below in this[1] commit.

![image](https://github.com/wso2/financial-open-banking/assets/9979373/60b52f2b-e613-47f5-92f9-af1488d40b88)

[1] - https://github.com/wso2/financial-open-banking/pull/1/commits/4b012dbd39fa03b1bfa95abde470b625203fec54

Reply:
Updated OBIAM and OBBI readme files as well